### ISSUE TITLE
All phonemes should have IPA entries phonemes file should have all of used ones

### ISSUE THREAD

**Issue by valdisvi** [espeak-ng/espeak-ng#769]
To improve organization of [phoneme definitions](https://github.com/espeak-ng/espeak-ng/blob/master/docs/phontab.md) files (`phonemes` and `ph_...`) in `phsource` folder should have [ipa](https://en.wikipedia.org/wiki/International_Phonetic_Alphabet) entries.
Should review all these files and add [ipa](https://github.com/espeak-ng/espeak-ng/blob/master/docs/phontab.md#ipa) entry, if it is missing.
```
phoneme a
  ipa æ
```
After that, phonemes which have defined only for some language, should be moved/copied to `phonemes` file that it has superset of all used IPA phonemes.

**Comment by Icenowy** [comment]
For IPA entries, how to mark for aspiration?

At least for Chinese (Mandarin/Cantonese), we use aspiration to differ `ptk` from `bdg` in romanized forms (both are voiceless, but the former are aspirated, voiced stop doesn't exist).

Currently the tS;h item in ph_zh adapts ipa, but it writes `tɕh` which could be wrongly interpreted.

There seems to be a sign `ʰ` that is available as a small `h` suitable for marking it as aspirated.

**Comment by valdisvi** [comment]
Yes, IPA symbol for aspiration is [ʰ](https://en.wikipedia.org/wiki/Aspirated_consonant).
